### PR TITLE
Add support for AlmaLinux 9

### PR DESCRIPTION
**Pull request by sharoninator**

### CODE REVIEW

**Comment by ramereth** [comment]
@sharoninator can you please rebase and resolve the conflicts?

File: test/integration/inspec/controls/default_spec.rb [hunk 1]
```diff
@@ -26,7 +26,11 @@
 
   %w(docker dockerd).each do |cmd|
     describe command "#{cmd} --version" do
+<<<<<<< HEAD
       its('stdout') { should match(/26.1/) }
+=======
+      its('stdout') { should match(/26\./) }
+>>>>>>> fc70022 (Added almalinux 9)
```
Comment (ramereth) [inline test/integration/inspec/controls/default_spec.rb:33]: This was missed when you did your rebase. Please fix and squash your commits.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — CHANGES_REQUESTED** [review]
Please add AlmaLinux 9 to spec tests.

**Comment by ramereth** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v4.14.1.  Have a nice day!

**Pull request merged**